test/test_train_mp_imagenet.py

```diff
@@ -81,7 +96,34 @@
     momentum=0.9,
     lr=0.1,
     target_accuracy=0.0,
+    persistent_workers=False,
+    prefetch_factor=16,
+    loader_prefetch_size=8,
+    device_prefetch_size=4,
+    num_workers=8,
+    host_to_device_transfer_threads=1,
 )
+'''
+Best config to achieve peak performance on v4-8
+
+  1. It is recommended to use this config in conjuntion with XLA_USE_BF16=1 Flag. 
+  2. Hyperparameters can be tuned to further improve the accuracy.
+'''
+OPTIMIZED_KWARGS = dict(
```

Comment:
This is great, thanks @chandrasekhard2

Reply:
NOTE as future reference: the configs that work best on v4-8 are not expected to work as well on other hardware configs (e.g. v4-128)

@chandrasekhard2 wdyt about re-naming your `OPTIMIZED_KWARGS` variable? See below:

`OPTIMIZED_KWARGS_v4_8 = dict(...`
`OPTIMIZED_KWARGS = OPTIMIZED_KWARGS_v4_8`

Reply:
I agree with renaming this to clarify the accelerator this was optimized for. If it was optimized for multiple v4 sizes, then we can name it `OPTIMIZED_KWARGS_v4`.

If we're going to start publishing optimized hyperparameters for specific accelerators, we should move them to some sort of config file system, e.g. [gin](https://github.com/google/gin-config).

The corresponding [Flax](https://github.com/google/flax/tree/main/examples/imagenet) and [TensorFlow](https://github.com/tensorflow/models/tree/master/official/vision) examples both implement something like this.